hookman/hookman_generator.py

```diff
@@ -711,6 +712,31 @@ def _generate_windows_body(hooks):
         f"    }}",
         f"",
         f"",
+        f"    HMODULE Load_DLL(const std::wstring& w_filename) {{",
+        f'        wchar_t *path_env = NULL;',
+        f'        rsize_t path_env_len = 0;',
+        f'        // Read PATH environment variable',
+        f'        _wdupenv_s(&path_env, &path_env_len, L"PATH");',
+        f'        std::wstring::size_type dir_name_size = w_filename.find_last_of(L"/\\\\");',
```

Comment:
What's `L"/\\\\"`? IIUC this is `/\\` (without escapes), if so why having two `\`?

Reply:
At this point, when the C++ code is generated it is needed `\\` but there is a `\` scape to each one.

Reply:
Ahhh! right.

In this case I suggest to use `rf' ...'` in the string to avoid having to escape things twice. 😉